.travis.yml

```diff
@@ -1,43 +1,40 @@
-sudo: false
+sudo: required
 
 language: java
 
+addons:
+  apt:
+    packages:
+      - wget
```

Comment:
Is this actually necessary? We've already been using wget in make_dependencies.sh and the [Travis documentation](https://docs.travis-ci.com/user/reference/trusty/) says wget is already installed.

Reply:
Sorry, it is an oversight...